### ISSUE TITLE
VMs not reboot persistent - kubelet not starting because of swap

### ISSUE THREAD

**Issue by twillert** [kairen/kube-ansible#146]
Hi,
thanks for sharing this!

After setting up a working cluster the individual VMs are not reboot safe. After a reboot the kubelet was not start because of swap being enabled. The playbooks does a 'swapoff -a' which is not persistent.

One way of diabling swap in a persistent way would be to create a SystemD service to turn it off before starting Kubelet. Like this:

```
# /etc/systemd/system/swapoff.service

[Unit]
Description=Swapoff, kubelet requirement
After=network.target
Before=kubelet.service

[Service]
Type=oneshot
ExecStart=/sbin/swapoff -a
RemainAfterExit=true

[Install]
WantedBy=multi-user.target
```

Where would be the best place in the code to implement this? I would be happy to give a pull request a shot.

Cheers,
Thomas

**Comment by kairen** [comment]
Hi @twillert, thank you for your report, I think you can implement this in `k8s-setup role`.

**Comment by kairen** [comment]
fixed #147 .
